test/test_geos_functions.jl

```diff
@@ -80,10 +80,23 @@ LibGEOS.destroyGeom(expected_)
 LibGEOS.destroyGeom(output_)
 
 # GEOSCoordSeqTest
-
 cs_ = LibGEOS.createCoordSeq(5, 3)
-@fact LibGEOS.getSize(cs_) --> 5
-@fact LibGEOS.getDimensions(cs_) --> 3
+@fact LibGEOS.getSize(cs_) --> 1
+@fact LibGEOS.getDimensions(cs_) --> 2
+@fact LibGEOS.getCoordinates(cs_) --> [5, 3]
+
+cs_2 = LibGEOS.createCoordSeq([5.0, 3.0])
+@fact LibGEOS.getSize(cs_2) --> 1
+@fact LibGEOS.getDimensions(cs_2) --> 2
+@fact LibGEOS.getCoordinates(cs_2) --> [5.0, 3.0]
+
+cs_3 = LibGEOS.createCoordSeq([[5.0, 3.0], [1.0, 2.0], [1.0, 3.0]])
```

Comment:
I don't think this works on julia v0.4. Can you change it to `Vector{Float64}[[5.0, 3.0], [1.0, 2.0], [1.0, 3.0]]` instead?